File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        int cn=0;
        int cp=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]>0){
                cp++;
            }
            else if(nums[i]<0){
                cn++;
            }
        }
        return Math.max(cp,cn);
        
        
    }
}

File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.py
class Solution(object):
    def maximumCount(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        cp=0
        cn=0
        for i in nums:
            if(i<0):
                cn+=1
            elif(i>0):
                cp+=1
        return max(cn,cp)
        